libs/designsystem/src/lib/components/page/page.component.spec.ts

```diff
@@ -111,6 +111,48 @@ describe('PageComponent', () => {
     await TestHelper.whenReady(ionContent);
   });
 
+  describe('when on ios-phone without top safe-area', () => {
+    it('should show the toolbar with the correct height', async () => {
```

Comment:
Do we need all the additional describes now? It could maybe be reduced to a structure like 
```
describe('toolbar with dynamic height') {
    it('should be correct height on ios-phone without top-safe-area')
    it('should be correct height on ios-phone with top-safe-area')
    it('should show correctly on non-ios-phone ')
    ...
}
```

Reply:
Fixed